test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -36,7 +36,7 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
-from tests.functional import TestFunctional, SERVER
+from tests.functional import TestFunctional, SERVER, RSC
```

Comment:
you are right, it's probably better to just import everything. I just didn't like my editor telling me that some of the imports are unused :)